corehq/apps/linked_domain/util.py

```diff
@@ -25,16 +26,20 @@ def can_user_access_release_management(user, domain, check_toggle=False):
     """
     if not user or not domain:
         return False
-    if domain_has_privilege(domain, RELEASE_MANAGEMENT) or domain_has_privilege(domain, LITE_RELEASE_MANAGEMENT):
-        return user.is_domain_admin(domain)
-    elif check_toggle:
-        return toggles.LINKED_DOMAINS.enabled(domain)
-    return False
 
+    has_access = domain_has_privilege(domain, RELEASE_MANAGEMENT) and user.is_domain_admin(domain)
+    if include_lite_version:
```

Comment:
Feel free to ignore: I think it's simpler to have code like:
```
if some_flag and <logic relevant to flag>:
    return True
```

Reply:
I'm going to merge this into the base branch as is, but am happy to talk more about this when you're back. I may not have a good model of what you are suggesting, but what I am thinking of sounds like a couple of verbose `if` statements.

Reply:
Eh, I think we just disagree on which style is simpler, don't worry about it.